.circleci/config.yml

```diff
@@ -141,7 +143,6 @@ jobs:
           command: |
             nix-build nix --argstr revision $CIRCLE_SHA1
             mkdir -p bin
-            cp result-*bin/bin/crio-* bin
```

Comment:
We still need this line to copy the statically build binaries to the correct target dir :)

Reply:
There shouldn't be any crio-* though right? that used to be crio-config, but now we just have crio-. Do other binaries have crio- prepended?

Reply:
Yes the static binaries :)